pycbc/live/snr_optimizer.py

```diff
@@ -181,6 +181,23 @@ def normalize_initial_point(initial_point, bounds):
     return (initial_point - bounds[:,0]) / (bounds[:,1] - bounds[:,0])
 
 
+def init_population_array(init, parameter_count):
+    # Make sure you're using a float array
+    popn = numpy.asarray(init, dtype=numpy.float64)
+
+    if (numpy.size(popn, 0) < 5 or
```

Comment:
I don't know why you need all these checks when the function is only used once or twice.